apps/root/src/pages/earn/home/components/one-click-migration-modal/confirm-migration.tsx

```diff
@@ -0,0 +1,284 @@
+      <ContainerBox gap={4} alignItems="center">
+        <Radio checked={selected} />
+        <Typography variant="h6Bold" color={({ palette }) => colors[palette.mode].typography.typo2}>
+          {strategy.guardian?.name}
+        </Typography>
+      </ContainerBox>
+      <ContainerBox gap={3} alignItems="center">
+        {!!strategy?.guardian?.links && (
+          <>
+            <ContainerBox gap={2} alignItems="center">
+              {strategy.guardian.links.twitter && (
+                <Link
+                  underline="none"
+                  target="_blank"
+                  href={getLogoURL(strategy.guardian.links.twitter)}
+                  color={({ palette: { mode } }) => colors[mode].typography.typo5}
+                >
+                  <TwitterIcon
+                    size={SPACING(6)}
+                    sx={({ palette }) => ({ color: colors[palette.mode].typography.typo5 })}
+                  />
+                </Link>
+              )}
+              {strategy.guardian.links.discord && (
+                <Link
+                  underline="none"
+                  target="_blank"
+                  href={getLogoURL(strategy.guardian.links.discord)}
+                  color={({ palette: { mode } }) => colors[mode].typography.typo5}
+                >
+                  <DiscordIcon
+                    size={SPACING(6)}
+                    sx={({ palette }) => ({ color: colors[palette.mode].typography.typo5 })}
+                  />
+                </Link>
+              )}
+              {strategy.guardian.links.website && (
+                <Link
+                  underline="none"
+                  target="_blank"
+                  href={getLogoURL(strategy.guardian.links.website)}
+                  color={({ palette: { mode } }) => colors[mode].typography.typo5}
+                >
+                  <GlobalIcon
+                    size={SPACING(6)}
+                    sx={({ palette }) => ({ color: colors[palette.mode].typography.typo5 })}
+                  />
+                </Link>
+              )}
+            </ContainerBox>
+            <DividerBorder2 orientation="vertical" flexItem />
+          </>
+        )}
+        <ContainerBox gap={2} alignItems="center">
+          <Typography variant="bodySemibold" display="flex" alignItems="center" gap={1}>
+            <FormattedMessage description="earn.one-click-migration.confirm-migration.fees" defaultMessage="Fees" />
+            <Tooltip
+              title={
+                <Grid container spacing={3}>
+                  {strategy.guardian?.fees?.map((fee) => (
+                    <Grid key={fee.type} item xs={6}>
+                      <FeeItem fee={fee} intl={intl} />
+                    </Grid>
+                  ))}
+                </Grid>
+              }
+            >
+              <ContainerBox>
+                <InfoCircleIcon fontSize="small" />
+              </ContainerBox>
+            </Tooltip>
```

Comment:
Is this 'promoted' style necessary for the release?
![Screenshot 2024-12-04 at 10 22 51 AM](https://github.com/user-attachments/assets/2bbe45fd-5f6d-493e-a4ae-29e7f4f22422)

Reply:
Yes and I missed it!